### PR TITLE
[APIs] fix torch.save/load/min/max/sigmoid/nn.Sequential/nn.GroupNorm/nn.functional.interpolate...

### PR DESCRIPTION
**Pull request by Vooolta**
### PR Docs

[doc PR](https://github.com/PaddlePaddle/docs/pull/5920)
### PR APIs

```bash
torch.nn.Sequential
torch.nn.GroupNorm
torch.utils.data.BatchSampler
torch.Size
torch.Sigmoid
# 这些API传入参数是*arg，**kwargs
torch.linalg.norm
torch.save
torch.load
torch.nn.functional.interpolate
# diff
torch.max
torch.min
```

### CODE REVIEW

File: paconvert/transformer/basic_transformer.py [hunk 1]
```diff
@@ -90,6 +91,8 @@ def visit_Attribute(self, node):
         for torch_package in TORCH_PACKAGE_LIST:
             if full_attr.startswith("%s." % torch_package):
                 torch_api = full_attr
+                if torch_api in ALIAS_MAPPING:
```
Comment (zhwesky2010) [inline paconvert/transformer/basic_transformer.py:94]: 在 import_transformer 中做这个识别更换吧。

这两个地方更换成标准的公开API
https://github.com/PaddlePaddle/PaConvert/blob/master/paconvert/transformer/import_transformer.py#L156C1-L181

Comment (Vooolta) [reply paconvert/transformer/basic_transformer.py]: 已修改

File: paconvert/api_matcher.py [hunk 2]
```diff
@@ -3537,3 +3537,21 @@ def generate_code(self, kwargs):
         )
         code = API_TEMPLATE.format(self.kwargs_to_str(kwargs))
         return code
+
+
+class LinalgNormMatcher(BaseMatcher):
```
Comment (zhwesky2010) [inline paconvert/api_matcher.py:3542]: 这个不可以直接GenericMatcher吗

https://github.com/PaddlePaddle/PaConvert#%E6%AD%A5%E9%AA%A43%E9%85%8D%E7%BD%AEjson

File: paconvert/api_mapping.json [hunk 3]
```diff
@@ -7689,20 +7689,24 @@
     ],
     "kwargs_change": {
       "input": "x",
-      "dim": "axes"
+      "dim": "axis"
```
Comment (zhwesky2010) [inline paconvert/api_mapping.json:7692]: infoflow 2023-06-06 16-28-52
这个API是axes

File: tests/test_nn_modules_GroupNorm.py [hunk 4]
```diff
@@ -0,0 +1,55 @@
+# Copyright (c) 2023 PaddlePaddle Authors. All Rights Reserved.
```
Comment (zhwesky2010) [inline tests/test_nn_modules_GroupNorm.py:1]: 别名不用重新加单测文件，在原公开API里面，加个case：test_alias() 即可

File: paconvert/api_mapping.json (hunk 3 above)
Comment (Vooolta) [reply paconvert/api_mapping.json]: 已修改

File: tests/test_nn_modules_GroupNorm.py (hunk 4 above)
Comment (Vooolta) [reply tests/test_nn_modules_GroupNorm.py]: 已修改

File: paconvert/api_matcher.py [hunk 5]
```diff
@@ -411,6 +411,13 @@ def get_paddle_nodes(self, args, kwargs):
         if "other" in new_kwargs:
             call_maximinimum = True
 
+        if (
```
Comment (zhwesky2010) [inline paconvert/api_matcher.py:414]: 这个分支应该挪到上面的 
```
if len(args) > 1 and isinstance(args[1], ast.Name):
            call_maximinimum = True
```
里面去

Comment (Vooolta) [reply paconvert/api_matcher.py]: 已修改

**Review by zhwesky2010 — APPROVED** [review]
LGTM
